supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -531,6 +531,17 @@ There is no functional difference between the first server that was installed an
 
 *See also*:
 
+[discrete]
+[[interpreted-code]]
+==== image:images/yes.png[yes] interpreted code (noun)
+*Description*: In Red Hat Enterprise Linux, _interpreted code_ is source code that is executed step by step, without prior transformations, by a language interpreter, or a language virtual machine.
```

Comment:
I know this was added during another peer review, but I'm guessing the last comma isn't correct. Correct me if I'm wrong, but I read that as that the code is executed step by step and not transformed first, and then it's done by either a language interpreter or a language virtual machine. If my understanding is correct, then that comma shouldn't be there

```suggestion
*Description*: In Red Hat Enterprise Linux, _interpreted code_ is source code that is executed step by step, without prior transformations, by a language interpreter or a language virtual machine.
```